controller/loadBasicPokemonDetails.js

```diff
@@ -1,26 +1,35 @@
 'use strict';
 
 let fs = require('fs'),
-    basicPokemonDetail = require('../models/BasicPokemonDetail.js'),
+    basicPokemonDetail = require('../models/basicPokemonDetail.js'),
     pokemonListPath = require('../resources/json/pokemonlist.json');
```

Comment:
The file is currently called BasicPokemonDetail.js(with an uppercase B). So please revert this change or rename the file too if you wish.